pkgs/tests.zig

```diff
@@ -1,12 +1,14 @@
 comptime {
-    // _ = @import("state-transition/src/lib.zig");
+    _ = @import("state-transition/src/lib.zig");
     // uncomment in #3
     // _ = @import("state-proving-manager/src/manager.zig");
     // uncomment after rebase / fixing preset issue
     // _ = @import("params/src/lib.zig");
-    _ = @import("cli/src/main.zig");
+    // _ = @import("cli/src/main.zig");
     // uncomment after rebase / fixing preset issue
     // _ = @import("types/src/lib.zig");
-    _ = @import("./node/src/node.zig");
-    _ = @import("./node/src/forkchoice.zig");
+    // _ = @import("./node/src/node.zig");
+    // _ = @import("./node/src/forkchoice.zig");
+    // _ = @import("./types/src/lib.zig");
+    // _ = @import("./node/src/node.zig");
```

Comment:
more broken tests I suppose ... I have to get to the bottom of this. This is related to the way we build modules, for sure, but I hope it doesn't uncover a compiler bug.